src/scripts/build.ts

```diff
@@ -18,7 +18,9 @@ export default function build(customUpdatedAt: string | undefined): void {
 
 	const buildSongs = createSongs(updatedAt);
 
-	writeJson(buildSongs.json);
+	buildSongs.json.then((jsonString) => {
+		writeJson(jsonString);
+	});
```

Comment:
What's the purpose of this change? From what I can see `buildSongs.json` is of type string so this code wouldn't work

Reply:
The format function from Prettier used within `createSongs()` now returns a `Promise<string>` after the new update to Prettier v3.


```ts
//Prettier - index.d.ts
export function format(source: string, options?: Options): Promise<string>;
```